blog/entries/migrations/0007_auto_20190904_2357.py

```diff
@@ -0,0 +1,22 @@
+# Generated by Django 2.2.4 on 2019-09-04 23:57
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('entries', '0006_auto_20190903_0111'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='tag',
+            name='entries',
+        ),
+        migrations.AddField(
+            model_name='entry',
+            name='entry_tags',
+            field=models.ManyToManyField(to='entries.Tag'),
```

Comment:
I'm not entirely sure what created this migration, but this doesn't look right. You may want to consider downloading a SQLLite client, connecting to your Django database, and making sure that everything looks right.